main/stream.c

```diff
@@ -1071,6 +1071,27 @@ struct ast_stream_topology *ast_stream_topology_create_resolved(
 		}
 	}
 
+	for (; i < AST_VECTOR_SIZE(&configured_topology->streams); i++) {
+		struct ast_stream *configured_stream = AST_VECTOR_GET(&configured_topology->streams, i);
+		struct ast_stream *pending_stream =
```

Comment:
pending_stream is unused and causing compile failures.

Reply:
Removed unused variable.